test/util/bspNet/waits.ts

```diff
@@ -143,6 +143,53 @@ export const waitForBspStored = async (api: ApiPromise, checkQuantity?: number)
   }
 };
 
+/**
+
+ * Waits for a MSP to respond to storage requests.
+ *
+ * This function performs the following steps:
+ * 1. Waits for a short period to allow the node to react.
+ * 2. Checks for the presence of a 'mspRespondStorageRequestsMultipleBuckets' extrinsic in the transaction pool.
+ * 3. Seals a block and verifies the presence of an 'MspRespondedToStorageRequests' event.
+ *
+ * @param api - The ApiPromise instance to interact with the blockchain.
+ * @param checkQuantity - Optional param to specify the number of expected extrinsics.
+ * @returns A Promise that resolves when a MSP has sent a response to storage requests.
+ *
+ * @throws Will throw an error if the expected extrinsic or event is not found.
+ */
+export const waitForMspResponse = async (api: ApiPromise, checkQuantity?: number) => {
```

Comment:
It's kinda weird that this is under `bspNet`. A refactor is likely in order.

Reply:
I put this here to avoid potential conflicts with Pablo.